packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -33,22 +38,19 @@ contract Colosseum is Initializable, Semver {
      *  4) CHALLENGER_TURN    → bisect()                            → ASSERTER_TURN
      *  5) CHALLENGER_TURN    → when isAbleToBisect() returns false → READY_TO_PROVE
      *  6) CHALLENGER_TURN    → on bisection timeout                → CHALLENGER_TIMEOUT
-     *  7) ASSERTER_TIMEOUT   → when proveFault() succeeds          → PROVEN
+     *  7) ASSERTER_TIMEOUT   → when proveFault() succeeds          → NONE
      *  8) ASSERTER_TIMEOUT   → on proving timeout                  → CHALLENGER_TIMEOUT
-     *  9) READY_TO_PROVE     → when proveFault() succeeds          → PROVEN
+     *  9) READY_TO_PROVE     → when proveFault() succeeds          → NONE
      * 10) READY_TO_PROVE     → on proving timeout                  → CHALLENGER_TIMEOUT
      * 11) CHALLENGER_TIMEOUT → challengerTimeout()                 → NONE
```

Comment:
In this case, the challenge is deleted, so it's probably more accurate to say it's deleted rather than in a NONE state.